gene_finder.py

```diff
@@ -40,13 +61,27 @@ def get_reverse_complement(dna):
 
         dna: a DNA sequence represented as a string
         returns: the reverse complementary DNA sequence represented as a string
+
+        I did not add any more doctests because each string contains all the
+        nucleotides, so two doctests with random strings is sufficient.
+
     >>> get_reverse_complement("ATGCCCGCTTT")
     'AAAGCGGGCAT'
     >>> get_reverse_complement("CCGCGTTCA")
     'TGAACGCGG'
     """
-    # TODO: implement this
-    pass
+    reversed_dna = dna[::-1]
+    comp_list = []
+
+    for nuc in range(0, len(reversed_dna)):
+        comp_nuc = get_complement(reversed_dna[nuc])
+        comp_list.append(comp_nuc)
+
```

Comment:
if you wanna get fancy with just one line of code :
return ''.join([get_complement(reversed_dna[nuc]) for nuc in dna[::-1]])